pkg/transport/handshake.go

```diff
@@ -110,23 +123,33 @@ func MakeSettlementHS(init bool) SettlementHS {
 		if _, err := io.ReadFull(conn, accepted); err != nil {
 			return fmt.Errorf("failed to read response: %w", err)
 		}
-		if accepted[0] == 0 {
+		switch hsResponse(accepted[0]) {
```

Comment:
That will break settlement between older and newer visors, correct?

Reply:
No, it shouldn't. 0 is still success and 1 is a generic error like it was before, I just added more values and added sends